.github/workflows/release-rc.yml

```diff
@@ -0,0 +1,54 @@
+name: Release rc Workflow
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run && github.event.workflow_run.conclusion == 'success' || true }}
+    environment: 'Release NPM'
```

Comment:
It seems like this name will appear in Jira - https://spryker.atlassian.net/jira/software/c/projects/HRZ/deployments?startDate=-3m&endDate=now

but I wonder how to mark it as a production one?

Reply:
But it's ok that all release workflows are marked as `Release NPM`, no? Or you want to find a better name?

Reply:
name is OK, I just wonder how it can be marked as production release :)